models/intermediate/int__activities_by_prospect.sql

```diff
@@ -11,16 +11,19 @@ with activities as (
 ), joined as (
 
     select
+        coalesce(visitors.source_relation, activities.source_relation) as source_relation,
```

Comment:
shouldn't we just use `activities.source_relation` here as it's part of the base model? If not, we should probably flip the coalesce order so `activities.source_relation` gets picked first.